client/theme.js

```diff
@@ -130,6 +131,10 @@ const baseThemes = {
         foreground: colors.black,
         background: grays.light,
         border: grays.middleLight
+      },
+      secondary: {
+        foreground: colors.black,
+        background: grays.mediumLight
```

Comment:
If `secondary` is its own complete object then does it need a `border` color?

If the only difference between `primary` and `secondary` (and potential `tertiary`) is the background color and the rest is the same then maybe we structure it a bit differently.

Reply:
in mobile Panel we don't have borders, I'll check though